RADWIMPS_react.html

```diff
@@ -0,0 +1,47 @@
+<html>
+    <head>
+        <title>RADWIMPS</title>
+        <!-- React Library -->
+        <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
+        <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
+        <script src="https://unpkg.com/babel-standalone@6/babel.min.js"></script>

```

Comment:
- 念の為バージョンを固定したほうがよろしいかと
- 特に development 版を使う理由がなければ production (minimised) のほうがいいかなと
- こちらも念の為ですが SRI を設定したほうがよいかと思います
    - 特に理由がなければ babel-standalone にも `crossorigin` 属性を当てましょう

```suggestion
        <script 
            src="https://unpkg.com/react@16.14.0/umd/react.production.min.js"
            integrity="sha384-N7y5SSAooNlIfb9H750GR82ufkn1JXJFaCjg8pmt+OZuKcZoTvTGfog4d4taG/cF"
            crossorigin
        ></script>
        <script
            src="https://unpkg.com/react-dom@16.14.0/umd/react-dom.production.min.js"
            integrity="sha384-j7WmMv3OO6n8pZRATOsaMVEdZcHpoaTBIika/l92YM2AkEex72QunlTQlgmu+pI8"
            crossorigin
        ></script>
        <script
            src="https://unpkg.com/babel-standalone@6.26.0/babel.min.js"
            integrity="sha384-VgZgZqjY9Fh+YsIGYp/Vzpbz9fdsC3TLwB4MrpOstd9QcaJdH/8aYw5vwisv8i8d"
            crossorigin
        ></script>
```

Reply:
あと suggestion では直していませんが若干バージョンが古いのが気になりました，最新バージョン (`react@17`, `@babel/standalone@7`) でも動作しそうならそちらにアップグレードを推奨します

Reply:
懸念点は特にないと思います。